src/lib/blockchain_snark/blockchain_snark_state.ml

```diff
@@ -139,12 +139,16 @@ let%snarkydef_ step ~(logger : Logger.t)
     in
     (t, previous_state_hash, previous_blockchain_proof_input, body)
   in
+
   let%bind `Success updated_consensus_state, consensus_state =
     with_label __LOC__ (fun () ->
         Consensus_state_hooks.next_state_checked ~constraint_constants
           ~prev_state:previous_state ~prev_state_hash:previous_state_hash
           transition txn_snark.supply_increase )
   in
+  let global_slot =
```

Comment:
Can I use the `global_slot` from the newly computed consensus_state @deepthiskumar